integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/test/java/com/thinkbiganalytics/ingest/TableRegisterSupportTest.java

```diff
@@ -120,6 +122,26 @@ public void testTableCreate() {
         }
     }
 
+    @Test
+    public void testTableCreateS3() {
+        ColumnSpec[] specs = ColumnSpec.createFromString("id|bigint|my comment\nname|string\ncompany|string|some description\nzip|string\nphone|string\nemail|string\ncountry|string\nhired|date");
+        ColumnSpec[] parts = ColumnSpec.createFromString("year|int\ncountry|string");
+        TableRegisterConfiguration conf = new TableRegisterConfiguration("s3a://testBucket/model.db/","s3a://testBucket/model.db/","s3a://testBucket/app/warehouse/");
+        TableRegisterSupport support = new TableRegisterSupport(connection, conf);
+        TableType[] tableTypes = new TableType[]{TableType.FEED, TableType.INVALID, TableType.VALID, TableType.MASTER};
+        for (TableType tableType : tableTypes) {
+            String ddl =
+                    support.createDDL("bar", "employee", specs, parts, "ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.OpenCSVSerde'", "stored as orc", "tblproperties (\"orc.compress\"=\"SNAPPY\")",
+                            tableType);
+            String location = StringUtils.substringBetween(ddl, "LOCATION '", "'");
+            if(tableType.toString().equalsIgnoreCase("master")){
```

Comment:
Enums can be compared using the == operator, for example: `tableType == TypeType.MASTER`.